partners.html

```diff
@@ -0,0 +1,43 @@
+---
+layout: partners
+title: "Howdy, partner!"
+intro: "Join us in building tools to secure open source."
+
+contact-heading: "Be our partner"
+contact-text: "Join us in building tools to secure open source."
+contact-cta-text: "Contact our team"
+contact-email-address: "partners@snyk.io"
+contact-email-subject: "Inquiry about partnership opportunities with Snyk"
+contact-email-body: "Please provide contact details, company information, and a description of the request."
+
+heading: "Our partners"
+
+partners:
+  - partner: "BitHound"
+    url: "https://www.bithound.io/"
+    logo: "https://res.cloudinary.com/snyk/image/upload/v1481724345/partners/bithound.png"
+    text: "Beyond security, dependencies also introduce quality risks, such as outdated, unused and deprecated dependencies. Use <a href='https://www.bithound.io/'>bitHound</a> to easily manage your dependencies and code quality."
+  - partner: "Nodesource"
+    url: "https://nodesource.com/"
+    logo: "https://res.cloudinary.com/snyk/image/upload/partners/nodesource.png"
+    text: "<a href='https://nodesource.com/'>NodeSource</a> is The Enterprise Node.js Company that provide Node.js support, training, and consulting, trusted by the world's biggest brands."
+  - partner: "Twistlock"
+    url: "https://www.twistlock.com/"
+    logo: "https://res.cloudinary.com/snyk/image/upload/partners/twistlock.png"
+    text: "<a href='https://www.twistlock.com/'>Twistlock’s</a> technologies are built for containers — from hardening images in development to protecting containers in production — everything we do fits seamlessly in the container environment."
```

Comment:
This is the place on the page that has 1st tense "everything *we* do..."
Suggest to replace "everything we do" --> "everything Twistlock does"